src/components/Calculator.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+function Operations() {
+  return (
+    <div className="operators">
+      <button type="button" className="symbol" id="divide">&divide;</button>
+      <button type="button" className="symbol" id="multiply">&times;</button>
+      <button type="button" className="symbol" id="minus">-</button>
+      <button type="button" className="symbol" id="minus">+</button>
+      <button type="button" className="symbol" id="equals">=</button>
+    </div>
+  );
+}
+
+function Numbers() {
+  return (
+    <div className="numberKeys">
+      <button type="button">7</button>
+      <button type="button">8</button>
+      <button type="button">9</button>
+      <button type="button">4</button>
+      <button type="button">5</button>
+      <button type="button">6</button>
+      <button type="button">1</button>
+      <button type="button">2</button>
+      <button type="button">3</button>
+      <button type="button" className="zero">0</button>
+      <button type="button" className="dot">.</button>
+    </div>
+  );
+}
+
+function GeneralOperations() {
+  return (
+    <div className="controls">
+      <button type="button">AC</button>
+      <button type="button">+/-</button>
+      <button type="button">%</button>
+    </div>
+  );
+}
+
+class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.result = 0;
+  }
+
+  render() {
+    return (
+      <main>
+        <section className="calculator">
+          <p className="results">{ this.result }</p>
+          <div className="gOpsBody">
+            <GeneralOperations />
+          </div>
+          <div className="numBody">
+            <Numbers />
+          </div>
+          <div className="sideOps">
+            <Operations />
+          </div>
+        </section>
```

Comment:
- [ ] We are expected to stick to the design. There seems to be a disparity in the placement of your numbers and the operations. They should be on the same line as we have in the design.

  Given design | Your design
  :---:|:---:
  ![design-1](https://user-images.githubusercontent.com/36542986/182619679-11a5dd48-3c6c-4209-b445-c12d0723ec11.jpg) | ![design-2](https://user-images.githubusercontent.com/36542986/182619670-1a6cea1d-ba39-4b17-afaa-27a264b527c7.jpg)

Reply:
@TedLivist Thanks for the feedback. I have fixed that now.